Dockerfile

```diff
@@ -1,19 +1,8 @@
 FROM node:10.11
+WORKDIR /opt/sinopia_profile_editor/
 
-ENV HTML /opt/sinopia_profile_editor/
-WORKDIR $HTML
-
-COPY package*.json $HTML
-# FIXME: should this be npm install with --production flag?
-RUN cd $HTML && npm install
-
-# FIXME: do we want to run grunt locally and copy only dist/ to docker image?
-COPY Gruntfile.js $HTML
-COPY source/ $HTML/source/
-RUN cd $HTML && npm run grunt-dev
-
-COPY dist/ $HTML
-COPY server.js $HTML
+# Everything that isn't in .dockerignore ships
+COPY . .
```

Comment:
is this copying everything into the `WORKDIR`? When we spin up the container, are the necessary files now copied in?

Reply:
I just redid the commands to pull the latest image from Dockerhub and it ran as expected with the correct HTML in the body. Not sure it was a timing issue with Dockerhub or some other problem with the image I pulled in conflicting with a prior image. I'm fine with merging this PR although there are still some outstanding comments left.